beacon_node/beacon_chain/src/block_verification.rs

```diff
@@ -282,7 +287,11 @@ pub enum BlockError<T: EthSpec> {
     /// If it's actually our fault (e.g. our execution node database is corrupt) we have bigger
     /// problems to worry about than losing peers, and we're doing the network a favour by
     /// disconnecting.
-    ParentExecutionPayloadInvalid { parent_root: Hash256 },
+    ParentExecutionPayloadInvalid {
+        parent_root: Hash256,
+    },
+    PublishError,
+    SlashableProposal,
```

Comment:
Prior to [this check](https://github.com/sigp/lighthouse/pull/4316/files#r1234724202) being removed/updated `SlashableProposal` could only be thrown in the block production flow so I think this deserves a rename maybe to just `Slashable`?

Reply:
Would you also mind adding a doc related to what we decide on peer scoring?

Reply:
0066e6e8631c3d1c290cd817e998242c27ae726d